build-system/release-tagger/test/create-release.test.js

```diff
@@ -192,7 +192,103 @@ test('create', async (t) => {
       },
     });
 
-  await createRelease('2107280123000', '2107210123000');
+  await createRelease('2107280123000', '2107210123000', 'beta');
```

Comment:
Should this function call and the one on line 291 include the `prerelease` argument?

Reply:
Nope, I think you're referring to utils.js createRelease